src/backend/storeManagers/zoom/user.ts

```diff
@@ -0,0 +1,152 @@
+      return
+    }
+    try {
+      await this.makeRequest(`${apiUrl}/li/loggedin`)
+      logInfo('User is authenticated with Zoom', LogPrefix.Zoom)
+      const username = 'Zoom User' // Placeholder for now
+      configStore.set('username', username)
+      return { username }
+    } catch (error) {
+      logError(['Error verifying Zoom login:', error], LogPrefix.Zoom)
+      return
+    }
+  }
+
+  public static async getCredentials(): Promise<ZoomCredentials | undefined> {
+    if (!isOnline()) {
+      logWarning('Unable to get credentials - app is offline', {
+        prefix: LogPrefix.Zoom
+      })
+      return
+    }
+    if (!existsSync(tokenPath)) {
+      logError('No Zoom token available', LogPrefix.Zoom)
+      return
+    }
+    try {
+      const token = readFileSync(tokenPath, { encoding: 'utf-8' })
+      if (!token) {
+        logError('Empty Zoom token file', LogPrefix.Zoom)
+        return
+      }
+      return { access_token: token }
+    } catch (error) {
+      logError(['Error reading Zoom token:', error], LogPrefix.Zoom)
+      return
+    }
+  }
+
+  public static logout() {
+    clearCache('zoom')
+    configStore.clear()
+    if (existsSync(tokenPath)) {
+      unlinkSync(tokenPath)
+    }
+    logInfo('Logging user out from Zoom', LogPrefix.Zoom)
+  }
+
+  public static async isLoggedIn(): Promise<boolean> {
+    const tokenExists = existsSync(tokenPath)
+    const isLoggedInStore = configStore.get_nodefault('isLoggedIn') || false
+
+    if (!tokenExists) {
+      if (isLoggedInStore) {
+        configStore.set('isLoggedIn', false)
+      }
+      return false
+    }
+
+    if (!isOnline()) {
+      logWarning('App offline, cannot verify Zoom login status', LogPrefix.Zoom)
+      return isLoggedInStore // Return current store status if offline
+    }
+
+    try {
+      await this.makeRequest(`${apiUrl}/li/loggedin`)
+      logInfo('User is authenticated with Zoom (API verified)', LogPrefix.Zoom)
+      if (!isLoggedInStore) {
+        configStore.set('isLoggedIn', true)
+      }
+      return true
+    } catch (error) {
+      logError(['Zoom API login verification failed:', error], LogPrefix.Zoom)
+      configStore.set('isLoggedIn', false)
+      this.logout() // Clear token if API verification fails
+      return false
+    }
+  }
+
+  public static async makeRequest(url: string) {
+    const credentials = await this.getCredentials()
+    if (!credentials) {
+      throw new Error('Not authenticated with Zoom')
+    }
+    const headers = {
+      Authorization: `Bearer ${credentials.access_token}`,
+      Accept: 'application/json'
+    }
+
+    const response = await axios
+      .get(url, { headers })
+      .catch((error: AxiosError) => {
+        logError(['Zoom API request failed:', error.message], LogPrefix.Zoom)
+        throw error
+      })
+    return response.data
+  }
+
+  public static getLoginUrl(): string {
+    return `${embedUrl}/login?li=lutris&return_li_token=true`
```

Comment:
Lets use `li=heroic` here

Reply:
Done, it looks cool now! 
<img width="1800" height="333" alt="image" src="https://github.com/user-attachments/assets/8fb53ccf-b091-4437-859b-a0ab357e5d51" />